scripts/time_plot.py

```diff
@@ -1255,7 +1255,7 @@ def handle_websock_event(handlerkey,*args):
             if (args[1] not in html_layoutsettings):
                 html_layoutsettings[args[1]]=copy.deepcopy(html_layoutsettings['default'])
             send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[args[1]]])+'"]'+','+str(html_layoutsettings[args[1]]['ncols'])+')',handlerkey)
-            send_websock_cmd('document.getElementById("scriptnametext").innerHTML="'+scriptnametext+'";',handlerkey)
+            send_websock_cmd('document.getElementById("scriptnametext").innerHTML="'+str(scriptnametext)+'";',handlerkey)
```

Comment:
Is scriptnametext a `bytes`? If so you should use `scriptnametext.decode()`. It's probably also better to convert it where you extract it, rather than every time you want to use it.

Reply:
I think it is already a string, just did it to be precautionary. I am as unsure as you are what it is, so better to keep it in.

Reply:
I'm saying rather apply the caution when you set it. That way you can always just assume it is a string, instead of living in fear that it might not be.